.github/workflows/build.yml

```diff
@@ -51,7 +51,7 @@ jobs:
           sudo apt-get -y install --no-install-recommends apt-utils dialog git iproute2 procps lsb-release
       - name: install tool chain
         run: |
-          sudo apt-get -y install libstdc++-8-dev clang-format-10 ccache
+          sudo apt-get -y install libstdc++-8-dev clang-format-10 ccache cargo rustc
```

Comment:
we should add the same to `.devcontainer/Dockerfile` and `docker/Dockerfile.testing`